addons/udes_security/models/ir_attachment.py

```diff
@@ -101,8 +102,34 @@ def create(self, vals_list):
         for vals in vals_list:
             if "name" in vals:
                 vals = self._check_contents(vals)
+            if "attachment" in vals and "attachment_filename" in vals:
+                processed_attachment_data = self._process_attachment_data(
+                    vals["attachment"], vals["attachment_filename"]
+                )
+                vals["attachment"] = processed_attachment_data
         return super().create(vals_list)
 
+    def _process_attachment_data(self, attachment_data, attachment_filename):
+        # Check if the attachment is an image based on its file extension
+        allowed_image_extensions = [".jpg", ".jpeg", ".png"]
```

Comment:
Apparently (at least according to Wikipedia) `.webp` images support EXIF, so we should probably include those here too.